static/app/components/assigneeSelector.spec.jsx

```diff
@@ -39,6 +40,12 @@ describe('AssigneeSelector', () => {
       name: 'J J',
       email: 'jj@example.com',
     });
+    USER_4 = TestStubs.Member({
+      id: '4',
+      name: 'Jane Doe',
+      email: 'janedoe@example.com',
+      team_slug: 'cool-team2',
```

Comment:
This user was added with a different team slug so that this user is not part of `TEAM_1`, and thus cannot be assigned to `GROUP_2` which is owned by `TEAM_1`